Adresses #1585 NTR renal medullary fibroblast

bvarner-ebi opened the pull request:
Adresses #1585 NTR renal medullary fibroblast

emquardokus commented:
@shawntanzk I have a question for you. If there is more than simply location (kidney interaction) that differentiates this type of fibroblast, would your modeling still hold up? Do we always ignore molecular markers? (Immune cells relies on molecular markers for differentiation of T and B cell types).

emquardokus commented:
@bvarner-ebi yes, I understand; I think one think that is difficult is if they exist in a data table, but may not be explicitly stated in words.  I'm just feeling all of this out in terms of thoughts on how to deal with questions I also receive.  I think what Shawn stated "(one would assume the term renal medullary fibroblast would represent all fibroblasts in the renal medulla)," Please continue :-) and Thank-you!

bvarner-ebi commented:
> I think one think that is difficult is if they exist in a data table, but may not be explicitly stated in words. I

@emquardokus, yes, I've started looking at tables more closely after you found and shared some on previous tickets :) 
There are cases where some genes/markers appeared in tables, but other cell types also expressed them, but at reduced levels. In those cases I'm hesitant to classify them as defining biomarkers.
I think adding them in comments is reasonable in those cases.

bvarner-ebi commented:
As always, thanks for the review and suggestions, @shawntanzk.
 
> 1. is there other types of kidney interstitial fibroblasts? If so what are they and should they be added?

I added 'renal cortical fibroblast' as well. 

> 2. could we edit kidney interstitial fibroblast to have an equiv class, that way we should be able to better model the requested term as just `EquivalentTo fibroblast and (part_of some "renal medulla interstitium")`
>    My preference would be to have kidney interstitial fibroblast be `EquivalentTo fibroblast and ('part of' some 'kidney interstitium')`

I implemented your suggestions.

dosumis commented:
> can I just add how beautiful it is to see the difference between the reasoned and unreasoned diff :D

That warms my heart.
